Makefile

```diff
@@ -41,7 +41,7 @@ CROSS_COMPILE = arm-none-eabi-
 BUILD_VERSION := $(shell TZ= date +v%y.%j.%H%M)
 
 # Warning options
-WARN = -Wall -Werror -Wdouble-promotion -Wfloat-conversion
+WARN = -Wall -Wdouble-promotion -Wfloat-conversion
```

Comment:
-Werror was removed for the errors in the submodule.
They could eventually be overridden on a per-target basis if wanted.